reactfire/firebase-debug.log

```diff
@@ -0,0 +1,55 @@
+[debug] [2020-02-07T01:39:18.184Z] ----------------------------------------------------------------------
```

Comment:
Can you please add this to .gitignore? I've been meaning to do that for a while

Reply:
Will do